core/src/test/java/org/infinispan/executors/BlockingTaskAwareExecutorServiceTest.java

```diff
@@ -88,9 +73,9 @@ public boolean isSatisfied() throws Exception {
    }
 
    private BlockingTaskAwareExecutorServiceImpl createExecutorService() {
-      return new BlockingTaskAwareExecutorServiceImpl(new ThreadPoolExecutor(1, 2, 60, TimeUnit.SECONDS,
-                                                                             new LinkedBlockingQueue<Runnable>(1000),
-                                                                             new DummyThreadFactory()),
+      return new BlockingTaskAwareExecutorServiceImpl("BlockingAwareTest-Controller", new ThreadPoolExecutor(1, 2, 60, TimeUnit.SECONDS,
```

Comment:
Please use the full test class name for the controller thread, it's better for log filtering. Also, some local variables would help keep the code to the left.